scripts/bootstrap.sh.in

```diff
@@ -9,6 +9,7 @@ if test -z "$(type -p)"; then set -o pipefail; fi
 
 VERBOSE=${VERBOSE:-0}
 LOGFILE=/var/log/metalk8s/bootstrap.log
+BOOTSTRAP_FILE=/etc/metalk8s/bootstrap.yaml
```

Comment:
This is the "Bootstrap" script, so maybe this variable could just be named `CONFIG_FILE` (or even `CONFIG_FILEPATH`)